src/test/java/net/sf/jabref/logic/formatter/CaseChangersTest.java

```diff
@@ -3,19 +3,20 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+/**
+ * Tests in addition to the general tests from {@link net.sf.jabref.logic.formatter.FormatterTest}
+ */
 public class CaseChangersTest {
 
     @Test
     public void testChangeCaseLower() {
-        Assert.assertEquals("", CaseChangers.TO_LOWER_CASE.format(""));
         Assert.assertEquals("lower", CaseChangers.TO_LOWER_CASE.format("LOWER"));
```

Comment:
Since this is a cleanup PR, could you please also move these test cases to separate test files for the respective CaseChanger (e.g. LowerCaseFormatterTests)